regression/cbmc/pointer-primitive-check-03/main.c

```diff
@@ -22,7 +22,8 @@ void main()
   // offset out of bounds
   char *p5 = malloc(10);
   p5 += 10;
-  __CPROVER_r_ok(p5, 1);
+  _Bool result = __CPROVER_r_ok(p5, 1);
+  __CPROVER_assert(!result, "should be false");
 
```

Comment:
This test only indirectly relates to the pointer primitive checks -- I'd put it into pointer-predicates/

Reply:
Done (though I have instead chosen `r_w_ok10`).